vulnerabilities/tests/test_data_source.py

```diff
@@ -7,330 +7,34 @@
-        self.repodir = os.path.join(self.tempdir, "advisory-db")
-
-    def mk_ds(self, **kwargs) -> GitImporter:
-        kwargs["working_directory"] = self.repodir
-        kwargs["create_working_directory"] = False
-        kwargs["remove_working_directory"] = False
-
-        ds = mk_ds(**kwargs)
-        ds._update_from_remote = MagicMock()
-        return ds
-
-    def test_file_changes_last_run_date_and_cutoff_date_is_None(self):
-
-        ds = self.mk_ds(last_run_date=None, cutoff_date=None)
-
-        with ds:
-            added_files, updated_files = ds.file_changes(
-                subdir="rust", recursive=True, file_ext="toml"
-            )
-
-        assert len(updated_files) == 0
-
-        assert set(added_files) == {
-            os.path.join(self.repodir, f)
-            for f in {
-                "rust/cargo/CVE-2019-16760.toml",
-                "rust/rustdoc/CVE-2018-1000622.toml",
-                "rust/std/CVE-2018-1000657.toml",
-                "rust/std/CVE-2018-1000810.toml",
-                "rust/std/CVE-2019-12083.toml",
-            }
-        }
-
-    def test_file_changes_cutoff_date_is_now(self):
-
-        ds = self.mk_ds(last_run_date=None, cutoff_date=datetime.datetime.now())
-
-        with ds:
-            added_files, updated_files = ds.file_changes(
-                subdir="cargo", recursive=True, file_ext="toml"
-            )
-
-        assert len(added_files) == 0
-        assert len(updated_files) == 0
-
-    def test_file_changes_include_new_advisories(self):
-
-        last_run_date = datetime.datetime(year=2020, month=3, day=29)
-        cutoff_date = last_run_date - datetime.timedelta(weeks=52 * 3)
-        ds = self.mk_ds(last_run_date=last_run_date, cutoff_date=cutoff_date)
-
-        with ds:
-            added_files, updated_files = ds.file_changes(
-                subdir="crates", recursive=True, file_ext="toml"
-            )
-
-        assert len(added_files) >= 2
-        assert os.path.join(self.repodir, "crates/bitvec/RUSTSEC-2020-0007.toml") in added_files
-        assert os.path.join(self.repodir, "crates/hyper/RUSTSEC-2020-0008.toml") in added_files
-        assert len(updated_files) == 0
-
-    def test_file_changes_include_fixed_advisories(self):
-        # pick a date that includes commit 9889ed0831b4fb4beb7675de361926d2e9a99c20
-        # ("Fix patched version for RUSTSEC-2020-0008")
-        last_run_date = datetime.datetime(
-            year=2020, month=3, day=31, hour=17, minute=40, tzinfo=datetime.timezone.utc
-        )
-        ds = self.mk_ds(last_run_date=last_run_date, cutoff_date=None)
-
-        with ds:
-            added_files, updated_files = ds.file_changes(
-                subdir="crates", recursive=True, file_ext="toml"
-            )
-
-        assert len(added_files) == 0
-        assert len(updated_files) == 1
-        assert os.path.join(self.repodir, "crates/hyper/RUSTSEC-2020-0008.toml") in updated_files
-
-
-class TestOvalImporter(TestCase):
```

Comment:
IMHO this would be best moved to a new test_oval.py and not just deleted

Reply:
I cannot approve this if we just delete all these tests above.

Reply:
I have secured oval importer tests in the file itself https://github.com/nexB/vulnerablecode/pull/817/files#diff-eb1cffdadef3eacc470900468af6cce0bdd6ae476bc9ae94ce512863c060f850R40, and just deleted the GitImporter tests that are not related with the latest importers.